src/tests/unit/deprecated_adapter_support_tests.ts

```diff
@@ -3,8 +3,9 @@ import { FormSubmission } from "../../core/drive/form_submission"
 import { Adapter } from "../../core/native/adapter"
 import * as Turbo from "../../index"
 import { DOMTestCase } from "../helpers/dom_test_case"
+import { assert } from "@open-wc/testing"
 
-export class DeprecatedAdapterSupportTest extends DOMTestCase implements Adapter {
+class DeprecatedAdapterSupportTest extends DOMTestCase implements Adapter {
```

Comment:
I think we can directly call mocha's `setup` and `teardown` functions here and get rid of `DOMTestCase`.

Reply:
Done!